src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Ball {
+    private String colour;
+    private int number;
+
+    public Ball(String colour, int number) {
+        this.colour = colour;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Colour: " + this.colour
```

Comment:
You have a method `getRandomColor()` (named without `u`). Make everywhere names without `u`.